docs/developers/submitting_changes.rst

```diff
@@ -27,7 +27,7 @@ successfully contribute to the project. If any of the terms used are unfamiliar
 to you please do a quick search and then ask any of the contributors for
 assistance.
 
-* Fork the Imageworks OpenColorIO repository
+* Fork the ASWF OpenColorIO repository
```

Comment:
Let's change this to `AcademySoftwareFoundation/OpenColorIO` also, and link it to the GH page for clarity.